generators/app/package.json

```diff
@@ -16,7 +16,7 @@
   "private": true,
   "devDependencies": {
     "del": "^2.2.1",
-    "gulp": "^3.9.1",
+    "gulp": "gulpjs/gulp#4.0",
```

Comment:
Will moving to gulp 4 force users of custom-build to update all their other gulp files in all other projects that are still in gulp 3 syntax?